SigningService/IPackagePartSigner.cs

```diff
@@ -1,9 +1,12 @@
 using System.IO.Packaging;
+using System.Security.Cryptography;
+using System.Threading.Tasks;
 
 namespace SigningService
 {
     internal interface IPackagePartSigner
     {
-        bool TrySign(PackagePart packagePart);
+        Task<bool> TrySignAsync();
```

Comment:
If we pass stream in the arg it is fine although we need to parse PE file twice if we do that